### ISSUE TITLE
Out of Stock Product (Simple)

### ISSUE THREAD

**Issue by mahaidery** [woocommerce/woocommerce#21088]
Hello,

All of the products were set to be sold only once, So I enable manage Stock option set QTY to 1 and disallow back orders. Surprisingly, 1 product was sold 6 times. Now, I can’t figure out how did this happen? Can you please help identify the issue or to get to the root of this?

Pls let me know what additional information you need.

Its an urgent and sensitive matter. I need your support.

Thanks

**Comment by manojhl** [comment]
I'm sorry, but we don't use GitHub issues for support, we use it exclusively for WooCommerce bugs, enhancements and feature requests. If you need support, you could try WooCommerce support forums on WordPress.org (https://wordpress.org/support/plugin/woocommerce).

**Comment by peterfabian** [comment]
Hi, 

I could not reproduce this problem myself with WooCommerce alone, so I don't think it's a bug. 

Ideally, we would need the steps to reproduce the problem, we would need to see your settings and see  your [System status report](https://docs.woocommerce.com/document/understanding-the-woocommerce-system-status-report/). 
It would be best if you could go through the steps listed in the [Self service guide](https://docs.woocommerce.com/document/woocommerce-self-service-guide/) to rule out possible conflicts with other plugins and/or themes. 

Thanks.

**Comment by rodrigoprimo** [comment]
Closing this issue as we were not able to reproduce the problem and not enough details were provided. @mahaidery, if after following Peter's recommendation, you believe you found a WooCommerce bug, please open another issue using the [bug report template](https://github.com/woocommerce/woocommerce/issues/new?template=Bug_report.md).
